public/app/features/dashboard-scene/conditional-rendering/ConditionalRenderingGroup.tsx

```diff
@@ -53,55 +66,89 @@ export class ConditionalRenderingGroup extends ConditionalRenderingBase<Conditio
-    if (this === rootGroup) {
-      this.getConditionalLogicRoot().setState({ rootGroup: ConditionalRenderingGroup.createEmpty() });
-    } else {
-      rootGroup.setState({ value: rootGroup.state.value.filter((condition) => condition !== this) });
-    }
-    this.getConditionalLogicRoot().notifyChange();
+  public removeItem(key: string) {
+    this.setStateAndNotify({ value: this.state.value.filter((condition) => condition.state.key !== key) });
   }
 
   public serialize(): ConditionalRenderingGroupKind {
     if (this.state.value.some((item) => item instanceof ConditionalRenderingGroup)) {
       throw new Error('ConditionalRenderingGroup cannot contain nested ConditionalRenderingGroups');
     }
+
     return {
       kind: 'ConditionalRenderingGroup',
       spec: {
+        outcome: this.state.outcome,
         condition: this.state.condition,
         items: this.state.value
           .map((condition) => condition.serialize())
           .filter((item) => item.kind !== 'ConditionalRenderingGroup'),
       },
     };
   }
+
+  public static createEmpty(): ConditionalRenderingGroup {
+    return new ConditionalRenderingGroup({ condition: 'and', outcome: 'show', value: [] });
+  }
 }
 
 function ConditionalRenderingGroupRenderer({ model }: SceneComponentProps<ConditionalRenderingGroup>) {
   const styles = useStyles2(getStyles);
-  const { condition, value } = model.useState();
+  const { condition, outcome, value } = model.useState();
+
+  const [viewNewRule, setViewNewRule] = useState(false);
+
+  useEffect(() => setViewNewRule(false), [value]);
+
+  const outcomeOptions: Array<SelectableValue<'show' | 'hide'>> = useMemo(
+    () => [
+      { label: t('dashboard.conditional-rendering.group.outcome.show', 'Show'), value: 'show' },
+      { label: t('dashboard.conditional-rendering.group.outcome.hide', 'Hide'), value: 'hide' },
+    ],
+    []
+  );
 
   const conditionsOptions: Array<SelectableValue<'and' | 'or'>> = useMemo(
     () => [
-      { label: t('dashboard.conditional-rendering.group.condition.meet-all', 'Meet all'), value: 'and' },
-      { label: t('dashboard.conditional-rendering.group.condition.meet-any', 'Meet any'), value: 'or' },
+      { label: t('dashboard.conditional-rendering.group.condition.all', 'And'), value: 'and' },
+      { label: t('dashboard.conditional-rendering.group.condition.any', 'Or'), value: 'or' },
     ],
     []
   );
 
+  const variables = sceneGraph.getVariables(model).state.variables;
+
+  const newRuleOptions: Array<SelectableValue<'data' | 'variable' | 'interval'>> = useMemo(
+    () => [
+      { label: t('dashboard.conditional-rendering.group.add.data', 'Query result'), value: 'data' },
+      {
+        label: t('dashboard.conditional-rendering.group.add.variable', 'Template variable'),
+        value: 'variable',
+        isDisabled: variables.length === 0,
+      },
+      {
+        label: t('dashboard.conditional-rendering.group.add.interval', 'Dashboard time range less than'),
```

Comment:
Is interval condition a time range (window) length check or a "how old is the start time off the range"  check? if it's a window range we should rename it to be more clear I think (code code and here. "Time range less than". 

I think we also could need a "Time range start older than"  and "Time range end older than"

Reply:
and ideally the input to all these should be Combobox with some standard intervals (And support for custom value)